src/main/scala/com/fulcrumgenomics/umi/CorrectUmis.scala

```diff
@@ -185,7 +185,10 @@ class CorrectUmis
         case Some(umi: String) =>
           val sequences = umi.split('-')
           if (sequences.exists(_.length != umiLength)) {
-            if (wrongLengthRecords == 0) logger.warning(s"Read (${rec.name}) detected with unexpected length UMI(s): ${sequences.mkString(" ")}")
+            if (wrongLengthRecords == 0) {
+              logger.warning(s"Read (${rec.name}) detected with unexpected length UMI(s): ${sequences.mkString(" ")}.")
+              logger.warning(s"Expected UMI length: ${umiLength}")
```

Comment:
Apologies for the unrelated change here.  I had a stupid type that took me far too long to figure out because I used `-u` instead of `-U` and `CorrectUmis` happily decided my filename was the sole UMI sequence to correct to.  It the message here had told me my expcted UMI length was 30+ that would have helped!